Userland/Utilities/readelf.cpp

```diff
@@ -311,7 +311,7 @@ int main(int argc, char** argv)
     RefPtr<ELF::DynamicObject> object = nullptr;
 
     if (elf_image.is_dynamic()) {
-        if (interpreter_path.is_null()) {
+        if (interpreter_path.is_null() || interpreter_path.is_empty()) {
```

Comment:
It seems like the is null check is useless then? Should we just get rid of it? 
```suggestion
        if (interpreter_path.is_empty()) {
```

Reply:
Ah will do, I wasn't sure about that as maybe there was a way to get `interpreter_path` to be null when PT_INTERP is present, but on rereading the code that doesn't seem to be possible.